Example/Screenshots/ViewController.swift

```diff
@@ -38,7 +30,18 @@ class ViewController: NSViewController {
   }
 
   @IBAction func didTapCreateScreenshot(sender: Any) {
-    cliScreenshots.createScreenshot()
+    cliScreenshots.createScreenshot { (result) in
+      switch result {
+      case .success(let screenshot):
+        DispatchQueue.main.async {
+          self.textField.stringValue = "Success rect: \(String(describing: screenshot.rect))"
```

Comment:
let's use guard statement to make sure that self exists in the block or use [weak self] in block

Reply:
@memstel fixed -> https://github.com/blackbeltlabs/Screenshots/pull/6/commits/42b439e69c810a0b624d6868ca905f49c80c1367